README.md

```diff
@@ -46,6 +46,25 @@ if you prefer to prebuild internal dependencies, add `--skip-internal` to the ab
 
 Most packages will require a .env file. `.env.example` files indicate the required variables per package.
 
+### Local database
```

Comment:
Perhaps this could fit in the database package, with a link from the root readme?

Reply:
Now that I replaced the documentation with a link, we can probably leave this here and don't link just a link from the database README? What do you think? Given that most people will look at the main README for the basics (setting up the DB etc)

Reply:
Sounds good 👍, I agree